clang/include/clang/Driver/Options.td

```diff
@@ -6911,6 +6911,18 @@ def fsycl_fp32_prec_sqrt : Flag<["-"], "fsycl-fp32-prec-sqrt">, Group<sycl_Group
   Visibility<[ClangOption, CC1Option]>,
   HelpText<"SYCL only. Specify that single precision floating-point sqrt is correctly rounded.">,
   MarshallingInfoFlag<CodeGenOpts<"SYCLFp32PrecSqrt">>;
+def fsycl_use_integration_headers :
```

Comment:
Looks like a good opportunity to use `BoolFOption` instead of 2 separate entries.

Reply:
Can you suggest how I can fit in all the properties when using BoolFOption?

Reply:
whoa - glitch in the matrix.  I'll respond above.

Reply:
There has been some updates to the options table - you may need to update your workspace.  With the updates I performed recently, all of the SYCL based options are grouped together in to a common `sycl_Group` entry.  Doing so allows for the `Group<sycl_Group>` to be dropped from your definition which should better match with the template.

Reply:
Thanks for this.   Done.